test/common/test-functions.js

```diff
@@ -5,13 +5,20 @@
  * for routine operations.
  */
 
-function ph_select(sel) {
+ph_set_frame = (name) => {
```

Comment:
The `ph` here stands for Phantom.  Maybe we could rename these to `test_set_frame` or similar.

Reply:
Right, I know. None of the functions in test-functions.js are Phantom specific, this is why I renamed the file to something neutral. Once this actually works, we can also rename the injected functions, but this doesn't need to happen as a first step.